package.json

```diff
@@ -28,7 +28,7 @@
     "@codesandbox/sandpack-react": "^1.17.0",
     "@codesandbox/sandpack-themes": "^1.17.0",
     "@floating-ui/react": "^0.21.1",
-    "@types/react": "17.0.40",
+    "@types/react": "18.3.3",
```

Comment:
why is this in `dependencies`? can we move this into `devDependencies`?

Reply:
Moved to dev deps